django/core/cache/backends/db.py

```diff
@@ -202,15 +209,35 @@ def _base_set(self, mode, key, value, timeout=DEFAULT_TIMEOUT):
                 return True
 
     def delete(self, key, version=None):
-        key = self.make_key(key, version=version)
-        self.validate_key(key)
+        self.delete_many([key], version)
+
+    def delete_many(self, keys, version=None):
+        key_list = []
+        for key in keys:
+            self.validate_key(key)
+            key_list.append(self.make_key(key, version))
+
+        self._base_delete_many(key_list)
+
+    def _base_delete_many(self, keys):
+        if not keys:
+            return
 
         db = router.db_for_write(self.cache_model_class)
         connection = connections[db]
-        table = connection.ops.quote_name(self._table)
+        quote_name = connection.ops.quote_name
```

Comment:
I am not sure this change is beneficial.

Reply:
It is used not only in the next line but also in the line 236.